### ISSUE TITLE
Duplicated contract awards

### ISSUE THREAD

**Issue by melish** [eaudeweb/scratch#209]
When there are multiple vendors for a contract award, they are imported as distinct award models:

http://prntscr.com/pd7s0e

In reality that is the same award and it would be better (for statistics) not to create duplicates

**Comment by alexneamtu** [comment]
Should we just make the name unique? What should we do about the contract value? Make a sum or just pick one (I see that in the example it's the same value, just wondering if it's always like that)?
The duplicates are coming from ted.

**Comment by melish** [comment]
Original award notice: https://ted.europa.eu/udl?uri=TED:NOTICE:497888-2018:TEXT:EN:HTML&src=0

having this structure in the XML:

![image](https://user-images.githubusercontent.com/1218168/65941736-65de2380-e434-11e9-9e39-3d43aedb0085.png)


There is only one `<OBJECT_CONTRACT>` and two `<AWARD_CONTRACT>`

For our purposes, the 'contract award' in scratch can be mapped to either of these two.

The first AWARD_CONTRACT has only one contractor, while the second has three contractors:

![image](https://user-images.githubusercontent.com/1218168/65941912-d2592280-e434-11e9-83b1-335f0901eff0.png)

The contractors must not be duplicated, they are 1:N (or N:N)  vs the contract award.

In other words, the last three rows in the table should be grouped:

![image](https://user-images.githubusercontent.com/1218168/65942070-2c59e800-e435-11e9-8a17-a431c9bc8638.png)
